cope with mupdf's newer upstream support for building a shared lib

Index: meson.build
--- meson.build.orig
+++ meson.build
@@ -19,8 +19,7 @@ zathura = dependency('zathura', version: '>=0.5.2')
 girara = dependency('girara-gtk3')
 glib = dependency('glib-2.0')
 cairo = dependency('cairo')
-mupdf = dependency('mupdf', required: false)
-mupdfthird = cc.find_library('mupdf-third')
+mupdf = cc.find_library('mupdf')
 
 build_dependencies = [
   zathura,
@@ -29,32 +28,19 @@ build_dependencies = [
   cairo,
 ]
 
-if not mupdf.found()
-  # normal build of mupdf
-  mupdf = cc.find_library('mupdf')
-  build_dependencies += [mupdf, mupdfthird]
-else
-  # build from Debian's libmupdf-dev
-  build_dependencies += [mupdf, mupdfthird]
+  build_dependencies += [mupdf]
 
   libjpeg = dependency('libjpeg')
   libjbig2dec = cc.find_library('jbig2dec')
   libopenjp2 = dependency('libopenjp2')
   gumbo = dependency('gumbo')
-  tesseract = dependency('tesseract')
-  leptonica = dependency('lept')
-  mujs = dependency('mujs')
 
   build_dependencies += [
     libjpeg,
     libjbig2dec,
     libopenjp2,
-    gumbo,
-    tesseract,
-    leptonica,
-    mujs
+    gumbo
   ]
-endif
 
 if get_option('plugindir') == ''
   plugindir = zathura.get_variable(pkgconfig: 'plugindir')
